new-lamassu-admin/src/pages/maintenance/MachineDetailsCard.js

```diff
@@ -0,0 +1,178 @@
+import { makeStyles } from '@material-ui/core/styles'
+import useAxios from '@use-hooks/axios'
+import classnames from 'classnames'
+import moment from 'moment'
+import React from 'react'
+import ActionButton from '../../components/buttons/ActionButton'
+import ConfirmDialog from '../../components/ConfirmDialog'
+import { Status } from '../../components/Status'
+import { ReactComponent as DownloadReversedIcon } from '../../styling/icons/button/download/white.svg'
+import { ReactComponent as DownloadIcon } from '../../styling/icons/button/download/zodiac.svg'
+import { ReactComponent as RebootReversedIcon } from '../../styling/icons/button/reboot/white.svg'
+import { ReactComponent as RebootIcon } from '../../styling/icons/button/reboot/zodiac.svg'
+import { ReactComponent as ShutdownReversedIcon } from '../../styling/icons/button/shut down/white.svg'
+import { ReactComponent as ShutdownIcon } from '../../styling/icons/button/shut down/zodiac.svg'
+import { ReactComponent as UnpairReversedIcon } from '../../styling/icons/button/unpair/white.svg'
+import { ReactComponent as UnpairIcon } from '../../styling/icons/button/unpair/zodiac.svg'
```

Comment:
Future refactor: we should create a svg module and import every svg from there, this is getting annoying =p